code/modules/vehicles/lawnmower.dm

```diff
@@ -0,0 +1,118 @@
+/obj/vehicle/ridden/lawnmower
+	name = "Doom. Co Ultra-Mega-Mower"
+	desc = "Equipped with reliable safeties to prevent <i>accidents</i> in the workplace. The safety light is <b>on</b>."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "lawnmower"
+	uses_integrity = 1
+	max_integrity = 200
+	var/emagged = FALSE
+	var/emagged_by = null
+	var/list/drive_sounds = list('sound/effects/mowermove1.ogg', 'sound/effects/mowermove2.ogg')
+	var/list/gib_sounds = list('sound/effects/mowermovesquish.ogg')
+	var/normal_variant = TRUE // This is just so the lawnmower doesn't explode twice on destruction and for initializing.
+
+
+/obj/vehicle/ridden/lawnmower/Initialize(mapload)
+	if(normal_variant)
+		. = ..()
+		AddElement(/datum/element/ridable, /datum/component/riding/vehicle/lawnmower)
+	else
+		. = ..()
+		AddElement(/datum/element/ridable, /datum/component/riding/vehicle/lawnmower/nukie)
+
+/obj/vehicle/ridden/lawnmower/emagged
+	emagged = TRUE
+	desc = "Equipped with reliable safeties to prevent <i>accidents</i> in the workplace. The safety light is off"
+
+/obj/vehicle/ridden/lawnmower/atom_destruction()
+	if(normal_variant)
+		explosion(src, -1, 1, 2, 4, flame_range = 3)
+		. = ..()
+	else
+		explosion(src, -1, 3, 5, 7, flame_range = 5)
+		. = ..()
+
+/obj/vehicle/ridden/lawnmower/on_emag(mob/user)
+	. = ..()
+	if(emagged)
+		to_chat(user, span_warning("The safety mechanisms on \the [src] are already disabled!"))
+		return
+	to_chat(user, span_warning("You disable the safety mechanisms on \the [src]."))
+	desc = "Equipped with reliable safeties to prevent <i>accidents</i> in the workplace. The safety light is <b>off</b>."
```

Comment:
Instead of setting the desc, use on_examine to append the safety light text to the end of the description.